Model/Cron.php

```diff
@@ -2300,4 +2270,17 @@ private function getVirtualStatus($status)
 
         return $status;
     }
+
+    /**
+     * Call the caseManagement helper function and log the passed message
+     *
+     * @param bool $autoCapture
+     * @param string $logComment
+     * @param ...$logValues
+     */
+    private function markPendingReviewAndLog(bool $autoCapture, string $logComment, ...$logValues): void
```

Comment:
What about calling these methods directly instead of adding this extra helper?

Reply:
I just wanted to group them together since I always want to log something when the `markCaseAsPendingReview` is called. I thought it would be a bit cleaner like this but again, I can remove it if you guys want.

Reply:
This API felt a bit strange to me. We are mixing business logic with logging which is something secondary. `CaseManagement` is a helper class with a logger, maybe we leave logging for `markCaseAsPendingReview`?

Reply:
That's a very good point. I included the logging in the business logic to ensure that each specific case will have some different info. I could've done this on this function (in Cron.php) but then I would have had to replicate the different scenarios (that are in the helper) in Cron.php, in order to be able to log the different messages

If Cron.php was in a good state I would have done this (or something similar) but due to it's current state I preferred to omit this for now.